packages/component-library/src/CardList/CardList.js

```diff
@@ -215,13 +156,16 @@ const CardList = ({ CardRegistry, tagsList, projects }) => {
                 {tagsList[category].sort().map(topic => (
                   <ListItem key={shortid.generate()} className={classes.nested}>
                     <Checkbox
-                      value={activeTags[topic]}
+                      value={filterStatus[category][topic]}
                       label={topic}
                       onChange={() => {
                         if (showAllStories) setShowAllStories(false);
-                        setActiveTags({
-                          ...activeTags,
-                          [topic]: !activeTags[topic]
+                        setFilterStatus({
+                          ...filterStatus,
+                          [category]: {
+                            ...filterStatus[category],
+                            [topic]: !filterStatus[category][topic]
+                          }
```

Comment:
The whole `onChange` function should be abstracted. As much as possible, we want to avoid doing JS in JSX